client/src/components/ContentNavigator/ContentDataProvider.ts

```diff
@@ -60,6 +60,8 @@ import {
   isContainer as getIsContainer,
 } from "./utils";
 
+const SAS_FILE_SEPARATOR = "~fs~";
```

Comment:
Fwiw, this is sas server specific. Anything that is specific to connection type should happen in the adapter if possible.